### PR TITLE
More Doc Fixes

### PR DESCRIPTION
**Pull request by YazzyYaz**

### CODE REVIEW

**Review by liamsi — APPROVED** [review]

File: docs/nodes/bridge-node.md [hunk 1]
```diff
@@ -150,7 +150,7 @@ Celestia Bridge Nodes
 Example:
 
 ```sh
-/ip4/46.101.22.123/tcp/2121/p2p/12D3KooWD5wCBJXKQuDjhXFjTFMrZoysGVLtVht5hMoVbSLCbV22
+/ip4/<ip-address>/tcp/2121/p2p/12D3KooWD5wCBJXKQuDjhXFjTFMrZoysGVLtVht5hMoVbSLCbV22
```
Comment (adlerjohn) [inline docs/nodes/bridge-node.md:153]: localhost? Or put it in a shell variable

Comment (YazzyYaz) [reply docs/nodes/bridge-node.md:153]: @adlerjohn was talking to @renaynay who has a preference of not putting local-host in most places over using <ip-address> instead.

Can go either way, just providing context

Comment (adlerjohn) [reply docs/nodes/bridge-node.md:153]: Okay, then shell variable would be preferred

Comment (YazzyYaz) [reply docs/nodes/bridge-node.md]: check latest commit and lemme know if that's what you want

**Review by adlerjohn — APPROVED** [review]

**Pull request merged**